### ISSUE TITLE
Field does not display?

### ISSUE THREAD

**Issue by sutcliffe121** [mostafaznv/nova-ckeditor#41]
I have followed the installation instructions but I don't see the field when I add it to a resource. There is an error in console when I load the resource with the field:

![image](https://user-images.githubusercontent.com/33236577/212480037-82757634-4251-4fc3-97d3-40b6c992270d.png)

**Comment by mostafaznv** [comment]
Hi @sutcliffe121 
thanks for the report.

Can you give me more explanation? (PHP, Laravel and package version. maybe your resource and config file?)
I couldn't reproduce this error.

**Comment by sutcliffe121** [comment]
PHP 8.1.2
Laravel 9.48
Nova 4.21
Package 3.4.1

I haven't made any changes to the config file. I published it as per the read me.

I added to my resource using:

use Mostafaznv\NovaCkEditor\CkEditor;

CkEditor::make('Content', 'blog')->stacked(),

When I load the resource, all fields load, but no ck editor. 

Console has this error:

![image](https://user-images.githubusercontent.com/33236577/213939587-2e8c3b0a-7ad0-4d96-af91-d06ca8f0358f.png)
